ui/app/pages/swaps/swaps.util.js

```diff
@@ -112,6 +112,10 @@ const QUOTE_VALIDATORS = [
     property: 'maxGas',
     type: 'number',
   },
+  {
+    property: 'gasEstimate',
+    type: 'number|undefined',
```

Comment:
Perhaps we could add a validator to ensure it's a positive, non-zero number?

```suggestion
    type: 'number|undefined',
    validator: (gasEstimate) => gasEstimate > 0,
```

I was about to suggest that your fallback multiplier use a `typeof` check instead of  `||` in case `0` was returned. But then realized that a multiplier of zero made no sense.